qldbdriver/statement_execution_test.go

```diff
@@ -448,6 +448,78 @@ func TestStatementExecution(t *testing.T) {
 		assert.True(t, ok)
```

Comment:
Are these real tests against actual production QLDB?

Reply:
That is correct. These are the integration tests along with `session_management_test.go`